themes/base.theme.bash

```diff
@@ -391,7 +391,17 @@ function rvm_version_prompt {
 function rbenv_version_prompt {
```

Comment:
Adding some tests to assure this wont break would be awesome. It totally saved us with the `$stash_count` change for example

Reply:
Are you referring to the whole function, or certain lines? You've marked only one line, so I'm not sure.

Reply:
I refer to the function, as it does some logic that can be tested

Reply:
Oh, I see. It shouldn't need it, as it's already being tested within the `while read` loop, plus it's pretty straightforward; I can't see it going wrong. It may, however, rely on BASH 4+, but I'm guessing _bash-it_ already has tests to that sort of effect.

Reply:
I see, it is indeed quite a small function, so it should be okay.
One problem with your approach is that we try to keep bash-it compatible with Bash3.2+, so we cant use Bash4+ only features :cry:  
What things that you use are bash4+ only?

Reply:
Oooh, damn. :( That's a real shame. Most of what I use make full use of modern BASH. I'll have to review this whole commit. Keeping _bash-it_ to such an old version is really going to hinder it's code and computational efficiency.

Why such a lower version? I understand the need for compatibility, especially in such a project, but I haven't even heard of a single person or use-case in which someone would use such an old version. Could I tempt you to push that to at least 4? Lol

Thank you for your time in looking into and replying to this commit, BTW.

Reply:
@terminalforlife it's because MacOS

Reply:
Aaah, I thought they'd especially be up-to-date, or is because of older Mac systems?

Reply:
@terminalforlife You would think, but every version before Catalina uses BASH 3.x. The way to install 4.x is through `brew`, editing `/etc/shells`, and changing shells with `chsh`. In Catalina they use `zsh` as the default shell, and while they ship `bash` in case a user is upgrading in-place, the version is still 3.x :man_facepalming:.

Reply:
@terminalforlife You would think, but every version before Catalina uses 3.x. The way to install 4.x is through `brew`, editing `/etc/shells`, and changing shells with `chsh`. In Catalina they use `zsh` as the default shell, and while they ship `bash` in case a user is upgrading in-place, the version is still 3.x :man_facepalming:.

P.S. My personal take on this topic is that bash-it should continue on the path we've set for cleanup through the rest of the year. Once everything is cleaned up (mostly for patterns and style), we could benefit from an internal refactor, which may or may not happen along the way. Once this is all done, we would be better set up to address this issue by detecting major version and loading optimized functions.

Reply:
Okidoke. Definitely ignore this PR then. Is there a way to disable it without deleting it? Thanks for all the information, and especially for all the patience involved. When or if y'all decide to refactor, I'll stop by to see if I can pitch in. :)